src/Map.js

```diff
@@ -66,6 +66,14 @@ export class Map extends KeyedCollection {
     });
   }
 
+  static ownEntries(obj) {
+    if (typeof obj !== 'object') {
+      return obj;
```

Comment:
Presumably this function is expecting to return a Map, but this may return something that is not a Map. Should this throw an error? Actually, we should expect `Reflect.ownKeys` to throw that error, so these lines are probably unnecessary.